Core/src/main/java/com/plotsquared/core/queue/QueueCoordinator.java

```diff
@@ -75,12 +75,26 @@ public QueueCoordinator(@Nullable World world) {
      * @param x chunk x coordinate
      * @param z chunk z coordinate
      * @return a new {@link ScopedQueueCoordinator}
+     * @deprecated Use {@link ScopedQueueCoordinator#getForChunk(int, int, int, int)}
      */
+    @Deprecated(forRemoval = true, since = "TODO")
     public ScopedQueueCoordinator getForChunk(int x, int z) {
+        return getForChunk(x, z, 0, 255);
```

Comment:
The world's min and max height (inclusive) instead of 0 & 255?

Reply:
Or are you leaving it 0 - 255 cause of deprecation? There's a similar case below.